src/queue.rs

```diff
@@ -93,10 +93,10 @@ impl<T: TaskCell + Send> LocalQueue<T> {
 
     /// Gets a task cell from the queue. Returns `None` if there is no task cell
     /// available.
-    pub fn pop(&mut self) -> Option<Pop<T>> {
+    pub fn pop(&mut self, need_retry: bool) -> Option<Pop<T>> {
```

Comment:
I think `need_steal` is a more accurate name

Reply:
Oh indeed

Reply:
Fixed, thanks!